rules/eslint/suggestions.js

```diff
@@ -0,0 +1,213 @@
+    // handled by Prettier
+    'no-floating-decimal': 'off',
+    'no-global-assign': 'error',
+    'no-implicit-coercion': ['error', { allow: ['!!'] }],
+    'no-implicit-globals': 'error',
+    'no-implied-eval': 'error',
+    // inline comments are useful
+    'no-inline-comments': 'off',
+    'no-invalid-this': 'error',
+    'no-iterator': 'error',
+    // handled by "no-labels" rule
+    'no-label-var': 'off',
+    'no-labels': 'error',
+    'no-lone-blocks': 'error',
+    'no-lonely-if': 'error',
+    'no-loop-func': 'error',
+    // too strict to use
+    'no-magic-numbers': 'off',
+    // handled by Prettier
+    'no-mixed-operators': 'off',
+    'no-multi-assign': 'error',
+    'no-multi-str': 'error',
+    // sometimes negated condition is more explicit
+    'no-negated-condition': 'off',
+    // nested ternary with parens and multilining are useful
+    'no-nested-ternary': 'off',
+    'no-new': 'error',
+    'no-new-func': 'error',
+    'no-new-object': 'error',
+    'no-new-wrappers': 'error',
+    'no-nonoctal-decimal-escape': 'error',
+    'no-octal': 'error',
+    'no-octal-escape': 'error',
+    'no-param-reassign': ['error', { props: false }],
+    'no-plusplus': ['error', { allowForLoopAfterthoughts: true }],
+    'no-proto': 'error',
+    // handled by "no-var" and "no-shadow" rules
+    'no-redeclare': 'off',
+    'no-regex-spaces': 'error',
+    // no restricted exports by default
+    'no-restricted-exports': 'off',
+    // no restricted globals by default
+    'no-restricted-globals': 'off',
+    // no restricted imports by default
+    'no-restricted-imports': 'off',
+    // no specific properties to restrict
+    'no-restricted-properties': 'off',
+    'no-restricted-syntax': [
+      'error',
+      {
+        selector: 'ForInStatement',
+        message: 'for..in loops are slower than Object.{keys,values,entries} and have their caveats.',
+      },
+      // From eslint-config-airbnb
+      // https://github.com/airbnb/javascript/blob/b4377fb03089dd7f08955242695860d47f9caab4/packages/eslint-config-airbnb-base/rules/style.js#L333
+      {
+        selector: 'LabeledStatement',
+        message: 'Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand.',
+      },
+      {
+        selector: 'WithStatement',
+        message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
+      },
+    ],
+    'no-return-assign': ['error', 'always'],
+    'no-return-await': 'error',
+    'no-script-url': 'error',
+    'no-sequences': 'error',
+    'no-shadow': 'error',
+    'no-shadow-restricted-names': 'error',
+    // ternaries are very useful
+    'no-ternary': 'off',
+    'no-throw-literal': 'error',
+    'no-undef-init': 'error',
+    // undefined is useful
+    'no-undefined': 'off',
+    // disabled by default; enable if necessary
+    'no-underscore-dangle': 'off',
+    'no-unneeded-ternary': 'error',
+    'no-unused-expressions': 'error',
```

Comment:
Is `chai-friendly` supposed to be installed manually per local config?

Reply:
Yeah, let's add it as well